google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/GoogleCloudApiSelectorPanel.java

```diff
@@ -124,6 +120,16 @@ private void createUIComponents() {
     modulesComboBox = new ModulesComboBox();
     modulesComboBox.fillModules(project);
 
+    ApplicationManager.getApplication()
+        .runReadAction(
+            () -> {
+              Module[] modules = ModuleManager.getInstance(project).getSortedModules();
+              if (modules.length > 0) {
```

Comment:
what happens if there are no modules?

Reply:
we should probably show some kind of message to the user in this case

Reply:
Can you have a project with no modules? I didn't think that was possible.

Reply:
ah yea - you can manually delete all modules; but you don't have a meaningful project at that point (no source etc.). so i guess its fine to just show an empty dropdown

Reply:
I'd make sure the "next" button is disabled though in this case; since I think you got rid of that override

Reply:
Done.